redirects.txt

```diff
@@ -215,3 +214,39 @@ applications/general/odoo_basics/users.rst applications/general/users.rst
 applications/general/odoo_basics/choose_language.rst  applications/general/users/language.rst             # odoo_basics/choose_language -> users/language
 
 applications/finance/sign/overview/signature_validity.rst applications/finance/sign.rst        #sign/overview/signature_validity -> sign/*
+
+services/support/supported_versions.rst                 administration/supported_versions.rst
+applications/general/auth/google_spreadsheets.rst       applications/sales/crm/performance/google_spreadsheets.rst
+developer/webservices/iap.rst                           developer/misc/api/iap.rst
+developer/webservices/odoo.rst                          developer/misc/api/odoo.rst
+developer/webservices/upgrade.rst                       developer/misc/api/upgrade.rst
+developer/webservices/localizations.rst                 developer/misc/i18n/localization.rst
+developer/reference/translations.rst                    developer/misc/i18n/translations.rst
+developer/reference/cmdline.rst                         developer/misc/other/cmdline.rst
+developer/reference/guidelines.rst                      developer/misc/other/guidelines.rst
+developer/reference/iot.rst                             developer/misc/other/iot.rst
+applications/general/multi_companies.rst                applications/general/users/companies.rst
+applications/general/unsplash.rst                       applications/websites/website/optimize/unsplash.rst
```

Comment:
Since you made a separate "section" for developer redirects, why don't you put the `dev/...` rules there ?

Reply:
ok, changed.